clang/test/3C/liberal_itypes_ptrptr.c

```diff
@@ -126,8 +126,8 @@ void nested_caller(void) {
   //CHECK: nested_callee_ptrptr(_Assume_bounds_cast<_Ptr<char *>>(nested_callee_ptrptr(_Assume_bounds_cast<_Ptr<char *>>(1))));
 }
 
-void itype_defined_ptrptr(int **p : itype(_Ptr<_Ptr<int>>)) {}
-//CHECK: void itype_defined_ptrptr(int **p : itype(_Ptr<_Ptr<int>>)) _Checked {}
+void itype_defined_ptrptr(int **p : itype(_Ptr<_Ptr<int>>));
+//CHECK: void itype_defined_ptrptr(int **p : itype(_Ptr<_Ptr<int>>));
```

Comment:
It's to keep the test testing the same thing. If it has the body, it can now be rewritten to a fully checked type, but the test was checking the behavior of function calls with itypes.